cmd/snap/cmd_userd_test.go

```diff
@@ -69,16 +69,15 @@ func (s *userdSuite) TestUserd(c *C) {
 		}()
 
 		needle := "io.snapcraft.Launcher"
-		for i := 0; i < 10; i++ {
+		for i := 0; i < 100; i++ {
```

Comment:
Could we make the end value conditional on `testing.Short()`?

Reply:
problem is, if you're using `-short` because you're on a slow machine, and when `testing.Short()` we wait less, ...